onnxscript/ir/_core.py

```diff
@@ -1187,18 +1189,18 @@ def __init__(
         self._version: int | None = version
         self._metadata: _metadata.MetadataStore | None = None
         self._metadata_props: dict[str, str] | None = metadata_props
-        self._graph: Graph | Function | None = graph
+        # _graph is set by graph.append
+        self._graph: Graph | None = None
```

Comment:
Where does self._graph get set to be =graph, does the setter for _graph get called somewhere else?